setup/index.js

```diff
@@ -62,8 +64,12 @@ async function downloadCLI() {
     }
   }
 
+  let os = platform;
+  if (os === 'win32') {
+    os = 'windows';
+  }
   // Download requested version
-  let asset = release.data.assets.find((a) => a.name.endsWith('_linux-amd64.tar.gz'));
+  let asset = release.data.assets.find((a) => a.name.endsWith(`_${os}-amd64.tar.gz`));
```

Comment:
If/When GitHub offer different architectures, we can update this to also use `process.arch` to detect the architecture, but this is not needed now